media-video/qtav/qtav-1.11.0.recipe

```diff
@@ -0,0 +1,152 @@
+HOMEPAGE="http://qtav.org/"
+COPYRIGHT="2013-2014 Wang Bin"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/wang-bin/QtAV/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="888675b2734940c693eec24f8b35a434a428f6827868876e149e0979d629bba5"
+SOURCE_DIR="QtAV-1.11.0"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qtav$secondaryArchSuffix = $portVersion
+	lib:libqtav$secondaryArchSuffix = 1.11.0 compat >= 1
+	lib:libqtavwidgets$secondaryArchSuffix = 1.11.0 compat >= 1
+
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	qt5$secondaryArchSuffix
+	lib:libass$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix >= 56
+	lib:libavdevice$secondaryArchSuffix >= 56
+	lib:libavfilter$secondaryArchSuffix >= 5
+	lib:libavformat$secondaryArchSuffix >= 56
+	lib:libpostproc$secondaryArchSuffix >= 53
+	lib:libavresample$secondaryArchSuffix >= 2
+	lib:libswresample$secondaryArchSuffix >= 1.1
+	lib:libswscale$secondaryArchSuffix >= 3
+	lib:libavutil$secondaryArchSuffix >= 54
+	lib:libgl$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libuchardet$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	qtav$secondaryArchSuffix = $portVersion
+	devel:libqtav$secondaryArchSuffix = 1.11.0 compat >= 1
+	devel:libqtavwidgets$secondaryArchSuffix = 1.11.0 compat >= 1
+
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}
+	qt5${secondaryArchSuffix}_devel
+	devel:libass$secondaryArchSuffix
+	devel:libavcodec$secondaryArchSuffix >= 56
+	devel:libavdevice$secondaryArchSuffix >= 56
+	devel:libavfilter$secondaryArchSuffix >= 5
+	devel:libavformat$secondaryArchSuffix >= 56
+	devel:libpostproc$secondaryArchSuffix >= 53
+	devel:libavresample$secondaryArchSuffix >= 2
+	devel:libswresample$secondaryArchSuffix >= 1.1
+	devel:libswscale$secondaryArchSuffix >= 3
+	devel:libavutil$secondaryArchSuffix >= 54
+	devel:libgl$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libuchardet$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	qt5${secondaryArchSuffix}_devel
+	devel:libass$secondaryArchSuffix
+	devel:libavcodec$secondaryArchSuffix >= 56
+	devel:libavdevice$secondaryArchSuffix >= 56
+	devel:libavfilter$secondaryArchSuffix >= 5
+	devel:libavformat$secondaryArchSuffix >= 56
+	devel:libpostproc$secondaryArchSuffix >= 53
+	devel:libavresample$secondaryArchSuffix >= 2
+	devel:libswresample$secondaryArchSuffix >= 1.1
+	devel:libswscale$secondaryArchSuffix >= 3
+	devel:libavutil$secondaryArchSuffix >= 54
+	devel:libgl$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libuchardet$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:find
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix
+	cmd:xargs
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	qmake ..
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+
+	mkdir -p $libDir
+	mkdir -p $includeDir/{QtAV/5.5.1/QtAV,QtAVWidgets}
```

Comment:
Why is the INSTALL rule not using make install? If that doesn't work, our qmake should be fixed to do the right thing there.
Otherwise, maybe a hack with qtconfig or pkgconfig or similar is possible?